MiniSim/Service/DeviceService.swift

```diff
@@ -425,6 +425,15 @@ extension DeviceService {
         try shellOut(to: "\(avdPath) delete avd -n \"\(device.name)\"")
     }
 
+    static func launchLogCat(device: Device) throws {
+        if device.booted {
```

Comment:
I think we already now that the device is booted because otherwise it won't be shown in the MenuBar